### PR TITLE
Update the unblock URL table on /educate/it

### PR DESCRIPTION
**Pull request by dju90**
[We received feedback](https://codedotorg.slack.com/archives/C61SCNESF/p1655862460762619) that the "Block" and "Unblock" sub-headings in the "Sites to Unblock" table on code.org/educate/it were confusing. We also got clearance from legal that the "To use YouTube hosted videos (deprecated late July 2018)" could be removed.

## Testing story
Tested on localhost

**Before**
image

**After**
image

## Follow-up work
- need to ensure that these changes are re-translated

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by davidsbailey — APPROVED** [review]
nice cleanup!

**Comment by MathyMcMatherson** [comment]
👋 Eavesdropping - based on [this Slack thread](https://codedotorg.slack.com/archives/C039X1ZLX/p1656525257523189), I wonder if Optimizely can be removed too? Thought I'd mention it while we're looking closely at this table...

1st Edit: Oh whoops - this is Dan Schneider from my non-Code.org Github account

2nd Edit: On closer look, based on that thread, I think [this support article](https://support.code.org/hc/en-us/articles/202591743-Technical-requirements-for-Code-org) will need to get updated too.

**Comment by davidsbailey** [comment]
good catch @dancodedotorg , it looks like it was removed in https://github.com/code-dot-org/code-dot-org/pull/47026 (but was showing here until a moment ago when Bryan merged in latest staging).

File: pegasus/sites.v3/code.org/public/educate/it.md [hunk 1]
```diff
@@ -72,10 +72,9 @@ For the very best experience with all Code.org content, we recommend consulting
...
-| **To use YouTube hosted videos** <br/>(Enhanced Privacy Mode - available late July 2018) | `https://youtube-nocookie.com/*`<br/>`https://*.googlevideo.com/*`<br/>`https://*.ytimg.com/*`                                                         |
-| **To use YouTube hosted videos** <br/>(Deprecated late July 2018)                        | `https://s.youtube.com/*`<br/>`https://www.youtube.com/*`<br/>`https://*.googlevideo.com/*`<br/>`https://*.ytimg.com/*`                                |
-| **To use Code.org hosted videos**                                                        | **Unblock:**<br/>`https://videos.code.org`<br/>**Block:**<br/>`https://www.youtube.com/favicon.ico`<br/>`https://www.youtube-nocookie.com/favicon.ico` |
+| **Optional**                                                                             | `https://cdn.optimizely.com`<br/>`https://www.google-analytics.com`<br/>`https://www.googleoptimize.com`                                               |
```
Comment (davidsbailey) [inline pegasus/sites.v3/code.org/public/educate/it.md]: hmm, @dju90 , actually now I am wondering if something went wrong with the merge. it looks like you may have undone what was done in https://github.com/code-dot-org/code-dot-org/pull/47056?

Comment (dju90) [reply pegasus/sites.v3/code.org/public/educate/it.md]: I think I fixed it...how's it look now?

**Comment by davidsbailey** [comment]
looks great!

**Pull request merged**